groups/rds/variables.tf

```
@@ -176,3 +176,21 @@ variable "rds_stop_schedule" {
   default     = ""
   description = "The SSM cron expression to define when the RDS instance should be stopped"
 }
+
+# ------------------------------------------------------------------------------
+# RDS CloudWatch Alarm Variables
+# ------------------------------------------------------------------------------
+variable "alarm_actions_enabled" {
+  type        = string
```

The type here should be `bool` (see the [module docs](https://github.com/companieshouse/terraform-modules/tree/main/aws/rds_cloudwatch_alarms)):

```suggestion
  type        = bool
```

---

Updated in latest commit